### PR TITLE
Add data source info for unit

### PR DESCRIPTION
**Pull request by vharvala**
Added `data_source` information for a unit.

### Before 

https://www.dropbox.com/s/8p75ww5jno62otx/Screenshot%202018-10-10%2015.05.53.png?dl=0

### After

https://www.dropbox.com/s/v1xx1xojucyfnil/Screenshot%202018-10-10%2015.06.24.png?dl=0

### CODE REVIEW

File: src/views/unit-details.coffee [hunk 1]
```diff
@@ -231,6 +231,7 @@ define (require) ->
                 _.max(x, (y) -> p13n.getTranslatedAttr(y.name)?.length)
 
             data.periods = @_serviceDetailsToPeriods data.services
+            data.data_source = data.data_source?.replace("www.", "")
```
Comment (tituomin) [inline src/views/unit-details.coffee]: Only from the beginning of the string maybe?
so /^www./
Although this wouldn't be a real problem, I suppose.

Comment (tituomin) [reply src/views/unit-details.coffee]: Actually, /^www\./

Comment (vharvala) [reply src/views/unit-details.coffee]: I think that it's a good suggestion. Even thought it wouldn't be a real problem, let's do it right anyway.

Comment (tituomin) [reply src/views/unit-details.coffee]: @vharvala still missing the backslash! :)

**Pull request merged**